charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/daemonset.yaml

```
@@ -33,4 +33,6 @@ spec:
       volumes:
         {{- toYaml .Values.adotCollector.daemonSet.volumes | nindent 10 }}
       serviceAccountName: {{ .Values.adotCollector.daemonSet.serviceAccount.name  }}
+      tolerations:
+        {{- toYaml .Values.adotCollector.daemonSet.tolerations | nindent 8 }}
```

```suggestion
        {{- toYaml .Values.adotCollector.daemonSet.tolerations | nindent 10 }}
```

I think `10` is the correct value here, based on what is used above for `volumes`.

---

I have confirmed that level 8 indentation iworks by using `helm template` and then applying the manifest generated
values
```
    tolerations:
      - key: "key1"
        operator: "Exists"
        effect: "NoSchedule"
```
manifest after template
```
          - hostPath:
              path: /dev/disk/
            name: devdisk
      serviceAccountName: adot-collector-sa
      tolerations:
        - effect: NoSchedule
          key: key1
          operator: Exists
```
daemonset spec
```
│ Tolerations:                 key1:NoSchedule op=Exists                                                                                                    │
│                              node.kubernetes.io/disk-pressure:NoSchedule op=Exists                                                                        │
│                              node.kubernetes.io/memory-pressure:NoSchedule op=Exists                                                                      │
│                              node.kubernetes.io/not-ready:NoExecute op=Exists                                                                             │
│                              node.kubernetes.io/pid-pressure:NoSchedule op=Exists                                                                         │
│                              node.kubernetes.io/unreachable:NoExecute op=Exists                                                                           │
│                              node.kubernetes.io/unschedulable:NoSchedule op=Exists                                                                        │
│ Events:                                                                              
```